scrunch/tests/mock_session.py

```diff
@@ -16,19 +16,27 @@ class FixtureAdapter(BaseAdapter):
     def __init__(self):
         self.fixtures = {}
         self.requests = []
+        self.post_responses = []
 
     def add_fixture(self, url, fixture):
         self.fixtures[url] = fixture
 
+    def add_post_response(self, response):
```

Comment:
`MockSession` didn't support POST requests, so had to add a few methods for the `create_subproject` test.